### ISSUE TITLE
Coordinator type tcp but serial port can be local only, should be user set for remote device socket/serial port too.

### ISSUE THREAD

**Issue by jussipo** [sasu-drooz/Domoticz-Zigate#1392]
When coordinator is tcp, also serial port should be editable, not local.
Example using fixed Lidl ZB gateway, serial port is remote and socket redirector is used, only way to fix is just force on plugin.py code:
Parameters["SerialPort"] = "socket://192.168.1.46:8888".
So maybe  editable serialport name?

**Comment by pipiche38** [comment]
That I a good point.
Would you mind proposing a pR ?

**Comment by pipiche38** [comment]
@jussipo Please switch to dev branch ( git checkout dev; git pull ) and make sure to restart Domoticz.

```
Coordinator Model:  Silicon Labs ...
Coordinator Type : Socket
IP: 192.168.1.46
Port: 8888
```

Available on 6.4.105
Commit: ddb6f323

**Comment by pipiche38** [comment]
@jussipo  could you let us know if this commit (integrated on dev branch) fixe the issue ?
